game/assets/jak1/subtitle/game_subtitle_en.gd

```diff
@@ -860,6 +864,10 @@
   (0 "BIRDWATCHER" "OH LOOK, A MUCKY-MUCK.")
   )
 
+("BIR-am08" :hint #x0
+  (0 "BIRDWATCHER" "HERE, BIRDIE BIRDIE...")
+  )
+
```

Comment:
duplicate

Reply:
yeah, it's weird, but it's also there in both upper- and lowercase: 

https://github.com/open-goal/jak-project/blob/944edd59386ef677824afa992364155b3c2c225f/goal_src/levels/beach/bird-lady.gc#L148

https://github.com/open-goal/jak-project/blob/944edd59386ef677824afa992364155b3c2c225f/goal_src/levels/beach/beach-obs.gc#L1101

no idea if `play-ambient` cares about the case or not. i actually didn't notice when adding that they were in two different files.

Reply:
Apparently it does matter... whudda thunk.